### ISSUE TITLE
UI broken in Resource DataStore tab

### ISSUE THREAD

**Issue by amercader** [ckan/ckan#7247]
**CKAN version**
2.10/master

**Describe the bug**
Missing icons and misaligned

![Screenshot 2022-11-28 at 11-38-11 dadada - 25_best_tacos_austin csv - CKAN](https://user-images.githubusercontent.com/200230/204535911-8e94acc1-7eed-4dc9-b7a1-d06891874274.png)

**Comment by amercader** [comment]
Probably a similar fix as https://github.com/ckan/ckan/pull/7169

**Comment by pdelboca** [comment]
A little bit more of context: The problem here is that the timeline depends on a css class `activity` that is no longer available in CKAN core since it has been moved to the `activity` extension.

The solution is to rebuild the html/css of the timeline to look similar to the activity stream. #7169 contains the `css` class removed and how to fix the icon display using Font Awesome stack (https://fontawesome.com/docs/web/style/stack).

This change should be done in datapusher extension (and if time, also xloader one.)

To summarise:
 - `datapusher` extension should contain all the CSS files required to display that log timeline properly (either by mimicking activity's CSS or building a new one)
 - `xloader` will have the same issue, but that should be fixed in its repository.
